<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>251</width>
      <height>201</height>
     </rect>
    </property>
    <property name="resizeMode">
     <enum>QListView::Adjust</enum>
    </property>
    <property name="layoutMode">
     <enum>QListView::SinglePass</enum>
    </property>
    <property name="viewMode">
     <enum>QListView::ListMode</enum>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>config file</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>browse</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>video</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_video"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>trajectory</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_trajectory"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_loadTrajectory">
             <property name="text">
              <string>load trajectory</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>3</number>
         </property>
         <widget class="QWidget" name="filter">
          <attribute name="title">
           <string>filter</string>
          </attribute>
          <widget class="QTableWidget" name="tableWidget">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>10</y>
             <width>256</width>
             <height>111</height>
            </rect>
           </property>
           <row>
            <property name="text">
             <string>1</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>2</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>1</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>2</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>3</string>
            </property>
           </column>
           <item row="0" column="0">
            <property name="text">
             <string>b</string>
            </property>
           </item>
           <item row="0" column="1">
            <property name="text">
             <string>a</string>
            </property>
           </item>
           <item row="0" column="2">
            <property name="text">
             <string>c</string>
            </property>
           </item>
           <item row="1" column="0">
            <property name="text">
             <string>e</string>
            </property>
           </item>
           <item row="1" column="1">
            <property name="text">
             <string>d</string>
            </property>
           </item>
           <item row="1" column="2">
            <property name="text">
             <string>f</string>
            </property>
           </item>
          </widget>
         </widget>
         <widget class="QWidget" name="search">
          <attribute name="title">
           <string>search</string>
          </attribute>
          <widget class="QPushButton" name="pushButton_2">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>10</y>
             <width>141</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>load search image</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>70</y>
             <width>171</width>
             <height>61</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>40</y>
             <width>81</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>load image</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="visulaze">
          <attribute name="title">
           <string>visulaze</string>
          </attribute>
          <widget class="QTableWidget" name="tableWidget_2">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>20</y>
             <width>231</width>
             <height>111</height>
            </rect>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="edit">
          <attribute name="title">
           <string>edit</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>659</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
